actions/crib-deploy-environment/action.yml

```diff
@@ -257,9 +269,9 @@ runs:
         fi
 
         echo "Creating ${DEVSPACE_NAMESPACE} Kubernetes namespace.."
-        kubectl create ns "${DEVSPACE_NAMESPACE}"
+        kubectl get ns "${DEVSPACE_NAMESPACE}" || kubectl create ns "${DEVSPACE_NAMESPACE}"
```

Comment:
The only thing that worries me is that maybe we should have a flag if we want to enable reuse of a namespace/override labels. Recently, we saw conflicts, such as the same UUID being generated multiple times. 

Anyway, it is gtg thanks for the improvements. Can you please just add a change set?